syntax ="proto3";
package llm;

service LlmService {
  rpc GetLlmAnswer(LlmRequest) returns (LlmReply);
}

message LlmRequest {
  string queryId =1;
  string query =2;
}

message LlmReply {
  string queryId=1;
  string answer=2;
}